contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -1866,6 +1866,7 @@ pltsql_store_view_definition(const char *queryString, ObjectAddress address)
 	uint64		flag_values = 0, flag_validity = 0;
 	char		*physical_schemaname;
 	const char  *logical_schemaname;
+	char *original_query = get_original_query_string();
```

Comment:
We should think if this can be done in better way. Having a global variable to track original query string is not most desirable.

Reply:
We are getting the original query from the Antlr and storing it in the `PLtsql_stmt_execsql` and need to pass it on to the hook `pltsql_store_view_definition` in hooks.c. 
Could you suggest something if not a global variable?